modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -115,7 +115,7 @@ function Question (props) {
               <List>
                 { labels.map( (item, idx) => {
                   return(
-                    <ListItem key={existingAnswer[0] + idx}> {defaultDisplayFormatter ? defaultDisplayFormatter(item, idx) : item} </ListItem>
+                    <ListItem key={existingAnswer[0] + idx}> {defaultDisplayFormatter ? defaultDisplayFormatter(item, idx, existingAnswer[1].value) : item}</ListItem>
```

Comment:
In DateQuestionUtilities.jsx, we appear to have some support implemented for date intervals. These would be stored as 2 values, so here `existingAnswer[1].value` would return an array. 

Additionally, I'm not a huge fan of changing the displayer signature by adding a new parameter. Instead, we should be able to access the value via the `idx` in specific question types directly.

```suggestion
                    <ListItem key={existingAnswer[0] + idx}> {defaultDisplayFormatter ? defaultDisplayFormatter(item, idx) : item}</ListItem>
```

Reply:
Fixed in [c0bf7ec](https://github.com/data-team-uhn/cards/pull/1879/commits/c0bf7ecabcf26671bb04cd5f532c9b76ba889180)